docs/essentials/reactivity-fundamentals.md

```diff
@@ -472,20 +472,20 @@ const { foo } = object
 ```
 
 ```vue-html
-{{ foo + 1 }}
+{{ '{{' }} foo + 1 {{ '}}' }}
 ```
 
 Now the render result will be `2`.
 
-One thing to note is that a ref will also be unwrapped if it is the final evaluated value of a text interpolation (i.e. a <code v-pre>{{ }}</code> tag), so the following will render `1`:
+One thing to note is that a ref will also be unwrapped if it is the final evaluated value of a text interpolation (i.e. a <code v-pre>{{ '{{' }} }}</code> tag), so the following will render `1`:
```

Comment:
I think this part should be as follows!
{{ '{{ }}' }}

Reply:
Thanks for finding this as well!